frontend/src/app/app.routes.ts

```diff
@@ -1,9 +1,19 @@
 import { Routes } from '@angular/router';
+import { MemberFormComponent } from './features/member/form/member-form.component';
+import { memberDataResolver } from './features/member/member-data-resolver';
 import { MemberOverviewComponent } from './features/member/overview/member-overview.component';
 import { MemberOverviewResolver } from './features/member/overview/member-overview-resolver';
 
-export const routes: Routes = [{
-  path: '',
+export const routes: Routes = [{ path: '',
   component: MemberOverviewComponent,
-  resolve: { filters: MemberOverviewResolver }
+  resolve: { filters: MemberOverviewResolver } },
+{
+  path: 'member',
```

Comment:
think about smth like this:

export const routes: Routes = [
  {
    path: 'member',
    children: [
      {
        path: '',
        component: MemberOverviewComponent
      },
      {
        path: 'add',
        component: MemberAddComponent
      }
    ]
  },
];

and then route per default to / member